numpy/core/fromnumeric.py

```diff
@@ -897,6 +898,13 @@ def sort(a, axis=-1, kind='quicksort', order=None):
     for the data type being sorted. It is currently mapped to
     merge sort.
 
+    .. versionadded:: 1.17.0
+    Timsort is added for better performance on already or nearly
+    sorted data. On random data timsort is almost identical to
+    mergesort. For details of timsort, refer to
+    `CPython listsort.txt <https://github.com/python/cpython/blob/3.7/Objects/listsort.txt>`_.
+
+
```

Comment:
Add a comment that timsort is used for stable sort, while quicksort is the default sort if none is chosen